meta-ivi/conf/distro/poky-ivi-systemd.conf

```diff
@@ -20,13 +20,13 @@ DISTRO_FEATURES_remove = "x11"
 DISTRO_FEATURES_BACKFILL_CONSIDERED = "sysvinit"
 
 # do not use gstreamer 1.2.3 by default
-PREFERRED_VERSION_gstreamer1.0              ?= "1.10.4"
```

Comment:
The commit comment talks about 1.8.3 but this is 1.10.4?  (no big deal though - we could let that slip through)